frontend/pages/docs/users/v-two-docs/reviewed/user/releases.mdx

```diff
@@ -0,0 +1,25 @@
+## **What is a Release?**
+
+A release is a different version of a model and there can be multiple released versions from the same model. Having a
+release gives you the opportunity to make alterations, e.g. updates, adding new features etc, without affecting or
+interfering with the original state of the model.
+
+## **How are the Releases reviewed?**
+
+Releases are reviewed by who the owner has chosen, to be the Model Technical Reviewer and the Model Senior Responsible
+Officer (ref to completing the model)
```

Comment:
Is this ref a todo? You should be able to link to specific parts of the documentation if that's your intention